packages/ckeditor5-list/src/listproperties/listpropertiesui.ts

```diff
@@ -334,14 +351,16 @@ function createListPropertiesView( {
 	dropdownView,
 	parentCommandName,
 	styleDefinitions,
-	styleGridAriaLabel
+	styleGridAriaLabel,
+	isStyleVisible
 }: {
 	editor: Editor;
 	propertiesConfig: Readonly<ListPropertiesConfig>;
 	dropdownView: DropdownView;
 	parentCommandName: string;
 	styleDefinitions: Array<StyleDefinition>;
 	styleGridAriaLabel: string;
+	isStyleVisible: boolean;
```

Comment:
This won't be needed if you pass a normalized config in the `propertiesConfig` parameter.